docs/en/observability/apm/agent-server-compatibility.asciidoc

```diff
@@ -10,11 +10,17 @@ The chart below outlines the compatibility between different versions of Elastic
 .1+|**APM AWS Lambda extension**
 |`1.x` |≥ `8.2`
 
+// Need APM integration version
+// Android
+.1+|**Android agent**
+|`0.x` |
+
 // Go
 .2+|**Go agent**
 |`1.x` |≥ `6.5`
 |`2.x` |≥ `6.5`
 
+// Update for 1.x?
 // iOS
 .1+|**iOS agent**
 |`0.x` |≥ `7.14`
```

Comment:
@bryce-b a couple questions about the [APM agent compatibility](https://observability-docs_3507.docs-preview.app.elstc.co/guide/en/observability/master/agent-server-compatibility.html) page: 

* What's versions of the APM integration are compatible with the Android agent?
* Should we also add a row for `1.x` for the mobile agents? If yes, what are the compatible APM integration versions?

Reply:
Yes, add a row for 1.x. 1.x will be compatible with 8.12

Reply:
While technically the Android agent is supported in previous versions, `8.12` is the first one where all the initial features are working (before that we had features marked as "Coming soon"), so I agree we should go with `8.12` to avoid any issues.

Now, regarding the agent version, we were planning to go GA with version `1.0` this month, however, I'm not sure if we're going to be able to release that version that soon as we need to work on a migration to another [base project](https://github.com/open-telemetry/opentelemetry-android) that's still not stable. So I'm a bit concerned about having to release version `1.0` this month and then in a couple more months having to release version `2.0` due to breaking changes added into OTel Android on its version `1.0`. I haven't discussed this yet with the mobile team, I was planning to do so on our weekly tomorrow, so for now, I'd leave it as `0.x`.

Reply:
I can confirm now that we're going to leave our Android agent version as `0.x` until its [base project](https://github.com/open-telemetry/opentelemetry-android) becomes stable.